app/models/concerns/campaigns/presentable.rb

```diff
@@ -20,6 +20,10 @@ def duration
       return nil if total_operative_days.zero?
       distance_of_time_in_words(start_date.beginning_of_day, end_date.end_of_day, scope: "datetime.distance_in_words.short")
     end
+    
+    def remaining_budget
+       total_budget.to_f - total_consumed_budget.to_f
+    end
```

Comment:
This logic already exists under `Campaigns::Budgetable#total_remaining_budget` https://github.com/gitcoinco/code_fund_ads/blob/master/app/models/concerns/campaigns/budgetable.rb#L17-L21